tests/e2e/standalone/workflow_test.go

```diff
@@ -0,0 +1,739 @@
+//go:build !windows && (e2e || template)
+
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package standalone_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	redisConnString = "--connection-string=redis://127.0.0.1:6379"
+)
+
+func TestWorkflowList(t *testing.T) {
```

Comment:
This test seems to be more focused on termination than list, and the list check is only a non-happy path.